### ISSUE TITLE
[Hello world]

### ISSUE THREAD

**Issue by ayomideadediran100-ai** [freeall/create-repository#3386]
this-repository-is-for-practicing-the-github-flow.

![jay3425 Banner Image](https://github.com/jay3425/jay3425/blob/main/0603(1).png)

  <h3 id="hey-there">Hey there <img src="https://github.com/jay3425/jay3425/blob/main/mylivewallpapers-com-Lofi-Computer-Boy-4K.mp4" width="25px"></h3>

<div>
  <img align="right" width="40%" src="https://github.com/jay3425/jay3425/blob/main/Dog%20Man%20GIF%20by%20Pluralsight.gif">
</div>

---

<img src="https://emojis.slackmojis.com/emojis/images/1531849430/4246/blob-sunglasses.gif?1531849430" width="30"/> ɪ'ᴍ Jay! 
*Digital Craftsman (Developer / Programmer)*
<br/> 

<p>
<a href="https://github.com/jay3425"><img src="https://img.shields.io/github/followers/jay3425?label=Follow&amp;style=social" alt="Github"></a></p>

               
<p align="left">I am a Data Scientist and Machine Learning Enthusiast with a huge love for Python, Java, RDBMS, Data Analysis, and Visualization. </p>

- ✨ Student of life :)
- 🧠 I see every project as a chance to learn, grow, and build something impactful.
- 🌱 I’m currently learning many things, I believe every day is a learning opportunity.
- 🚀 From Python to Pandas and HTML to TensorFlow, I love turning ideas into code and data into insight.
- 📊 Exploring the intersection of analytics, AI, and visualization—one dataset at a time.
- 💻 Visit my [Portfolio]() for more details about me.

<p align="left">
  <img src="https://komarev.com/ghpvc/?username=jay3425&label=Profile%20views&color=770677&style=for-the-badge&logo=star" alt="jay3425" style="padding-right:20px;" />
</p>

---

  
<center>
<h2 align="center">Tᴇᴄʜ sᴛᴀᴄᴋ</h2> </center>
<div class="badges">
  <img src="https://img.shields.io/badge/java-%23ED8B00.svg?style=for-the-badge&logo=openjdk&logoColor=white" alt="Java" />
  <img src="https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54" alt="Python" />
  <img src="https://img.shields.io/badge/r-%23276DC3.svg?style=for-the-badge&logo=r&logoColor=white" alt="R" />
  <img src="https://img.shields.io/badge/html5-%23E34F26.svg?style=for-the-badge&logo=html5&logoColor=white" alt="HTML5" />
  <img src="https://img.shields.io/badge/mysql-4479A1.svg?style=for-the-badge&logo=mysql&logoColor=white" alt="MySQL" />
  <img src="https://img.shields.io/badge/Matplotlib-%23ffffff.svg?style=for-the-badge&logo=Matplotlib&logoColor=black" alt="Matplotlib" />
  <img src="https://img.shields.io/badge/numpy-%23013243.svg?style=for-the-badge&logo=numpy&logoColor=white" alt="NumPy" />
  <img src="https://img.shields.io/badge/pandas-%23150458.svg?style=for-the-badge&logo=pandas&logoColor=white" alt="Pandas" />
  <img src="https://img.shields.io/badge/scikit--learn-%23F7931E.svg?style=for-the-badge&logo=scikit-learn&logoColor=white" alt="scikit-learn" />
  <img src="https://img.shields.io/badge/TensorFlow-%23FF6F00.svg?style=for-the-badge&logo=TensorFlow&logoColor=white" alt="TensorFlow" />
  <img src="https://img.shields.io/badge/github-%23121011.svg?style=for-the-badge&logo=github&logoColor=white" alt="GitHub" />
</div>

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="./Skills_Animation_Dark.gif">
  <source media="(prefers-color-scheme: light)" srcset="./Skills_Animation_White.gif">
  <img align="left" alt="GIF description" src="./Skills_Animation_White.gif">
</picture>
<br />

<h3 align="left">Current Learning</h3>
<ul align="left">
  <li>🤖 Advancing in Artificial Intelligence (AI) and Machine Learning (ML), with hands-on work on model development and evaluation.</li>
  <li>🧠 Exploring various Neural Network architectures, including CNNs and RNNs, to strengthen deep learning fundamentals.</li>
  <li>🗃️ Sharpening my skills in SQL for efficient data querying, manipulation, and database design.</li>
  <li>📊 Improving proficiency in Microsoft Excel, focusing on formulas, pivot tables, and data analysis tools.</li>
  <li>📈 Learning Power BI to design insightful dashboards and perform interactive data visualization.</li>
</ul>
  

<br />
<br />
<br />
<br />
<br />

   
<h2 align="center">🏆 Gɪᴛʜᴜʙ Tʀᴏᴘʜɪᴇs 🏆</h2>
<p align="center">
  <a href="https://github.com/jay3425">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://github-profile-trophy.vercel.app/?username=jay3425&no-bg=true&row=2&column=6&margin-w=20&margin-h=20&theme=monokai">
      <source media="(prefers-color-scheme: light)" srcset="https://github-profile-trophy.vercel.app/?username=jay3425&no-bg=true&row=2&column=6&margin-w=20&margin-h=20">
      <img alt="GitHub Trophies" src="https://github-profile-trophy.vercel.app/?username=jay3425&no-bg=true&no-frame=true&row=2&column=6&margin-w=20&margin-h=20">
    </picture>
  </a>
</p>

<br />

 
<h2 align="center">📊 Gɪᴛʜᴜʙ Sᴛᴀᴛs 📊</h2>

<table width="100%">
  <tr>
    <td width="50%">
      <h3 align="center"><strong>Gɪᴛʜᴜʙ Sᴛᴀᴛs</strong></h3>
      <p align="center">
        <a href="https://github.com/jay3425">
          <img align="center" src="https://github-readme-stats.vercel.app/api?username=jay3425&count_private=true&show_icons=true&theme=nightowl&bg_color=0,000000,441350&title_color=c56a90&text_color=ffffff&rank_icon=github&hide=prs,issues,contribs&show=reviews,prs_merged,prs_merged_percentage" alt="GitHub Stats" />
        </a>
      </p>
    </td>
    <td width="50%">
      <h3 align="center"><strong>Sᴛʀᴇᴀᴋ Sᴛᴀᴛs</strong></h3>
      <p align="center">
        <a href="https://github.com/jay3425">
          <img align="center" src="https://streak-stats.demolab.com?user=jay3425&theme=nightowl&background=0,000000,441350&fire=ffeb95&ring=ffeb95&sideNums=ffffff&sideLabels=ffffff&dates=c56a90&currStreakNum=ffffff" alt="Streak Stats" />
        </a>
      </p>
    </td>
  </tr>
  <tr>
    <td width="50%">
      <h3 align="center"><strong>Lᴀᴛᴇsᴛ Pʀᴏᴊᴇᴄᴛ</strong></h3>
      <p align="center">
        <a href="https://github.com/jay3425/Awesome-Dev-Portfolios">
          <img align="center" width="470" src="https://github-readme-stats.vercel.app/api/pin/?username=jay3425&repo=Awesome-Dev-Portfolios&theme=nightowl&show_owner=true&bg_color=0,000000,441350&title_color=c56a90&text_color=ffffff" alt="Awesome-Dev-Portfolios" />
        </a>
      </p>
    </td>
    <td width="50%">
      <h3 align="center"><strong>Tᴏᴘ Cᴏɴᴛʀɪʙᴜᴛɪᴏɴs</strong></h3>
      <p align="center">
        <a href="https://github.com/jay3425">
          <img align="center" src="https://github-contributor-stats.vercel.app/api?username=jay3425&limit=2&theme=nightowl&show_owner=true&combine_all_yearly_contributions=false&bg_color=0,000000,441350&title_color=c56a90&text_color=ffffff" alt="Top Repo" />
        </a>
      </p>
    </td>
  </tr>
</table>
<br />

<h2 align="center">📈 Cᴏɴᴛʀɪʙᴜᴛɪᴏɴ Gʀᴀᴘʜ 📈</h2>
<div align="center">
    <img src="https://github-readme-activity-graph.vercel.app/graph?username=jay3425&bg_color=220a28&&color=ffffff&line=c56a90&point=ffeb95&area=false&hide_border=false" border-radius="15">
</div>

---

 
<h2 align="center">🌟 Tʜᴏᴜɢʜᴛ ᴏғ ᴛʜᴇ Dᴀʏ 🌟</h2>

<p align="center">
    <img src="https://readme-daily-quotes.vercel.app/api?author=James%20Allen&quote=As%20a%20man%20thinketh%20in%20his%20heart%2C%20so%20shall%20he%20be.&theme=dark&bg_color=220a28&author_color=ffeb95&accent_color=c56a90">
</p>

 

<h2 align="center">🤝 Cᴏɴɴᴇᴄᴛ Wɪᴛʜ Mᴇ 🤝 </h2>
<div align="center">
  
<a href="mailto:jaydengle2005@gmail.com" target="_blank">
<img src="./gmail.png" width=50 height=50 alt="jaydengle2005@gmail.com" style="margin-bottom: 5px;" />
</a>

<a href="" target="_blank">
<img src="./instagram.png" width=50 height=50 alt="kiran_a_n" style="margin-bottom: 5px;" />
</a>

<a href="https://www.githubcom/jay3425" target="_blank">
<img src="./github.png" width=50 height=50 alt="Kiran1689" style="margin-bottom: 5px;" />
</a>

<a href="https://www.linkedin.com/in/jay-anil-dengle-049952337/" target="_blank">
<img src="./linkedin.png" width=50 height=50 alt="linkedin" style="margin-bottom: 5px;" />
</a>

</div>
<br/>

<div align="center">
<a href="https://www.buymeacoffee.com/jay3425" target="_blank"><img src="https://cdn.buymeacoffee.com/buttons/v2/default-yellow.png" alt="Buy Me A Coffee" style="height: 40px !important;width: 200px !important;" ></a>
</div>

 
<p align="center">
  <img src="https://capsule-render.vercel.app/api?type=waving&color=gradient&height=65&section=footer"/>
</p>
